.changeset/seven-dolphins-breathe.md

```diff
@@ -0,0 +1,5 @@
+---
+"@firebase/firestore": patch
+---
+
+The format of some of the IndexedDB data changed. This increases the performance of document lookups after an initial migration. If you do not want to migrate data, you can call `clearIndexedDbPersistence()` before invoking `enablIndexedDbPersistence()`.
```

Comment:
Typo in line 5, needs another 'e', should be enableIndexedDbPersistence().

Reply:
@egilmorez ... Talked to sebastian, and these changes are internal-only, no user-breaking effects. Thanks for adding me to the review.

@schmidt-sebastian ... Please feel free to include me in Firestore PRs, all platforms, especially the PRs that touch public-facing reference docs. Thanks!